compose.yaml

```diff
@@ -1,5 +1,11 @@
 services:
-
+  backend:
+    build:
+      context: .
+    ports:
```

Comment:
As stated in the Discord private call, we need `compose profiles` to manage when the `backend` service is started.
Starting it all the time will lead to Port Conflicts when starting the local application (both use 8080). 

We do not intend to run the dockerized `backend` locally.
Let's condition the `backend` service to a `production` profile.

https://docs.docker.com/compose/how-tos/profiles/